subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/ElasticLog.kt

```diff
@@ -0,0 +1,105 @@
+package com.avito.android.elastic
+
+import com.google.gson.FieldNamingPolicy
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import okhttp3.logging.HttpLoggingInterceptor.Level.BODY
+import retrofit2.Retrofit
+import retrofit2.converter.scalars.ScalarsConverterFactory
+import retrofit2.create
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatter.ISO_DATE
+import java.util.concurrent.TimeUnit.SECONDS
+
+/**
+ * @param endpoints list of elastic endpoints to send logs
+ * @param indexPattern see https://www.elastic.co/guide/en/kibana/current/index-patterns.html
+ * @param verboseHttpLog verbose http for debugging purposes
+ * @param onError can't deliver message; reaction delegated to upstream
+ */
+class ElasticLog(
```

Comment:
I want to see `ElasticLogger` with simple methods from interface `Logger`: debug(), info() ... etc

Reply:
it is not a logger, only transport, you don't need to invoke debug/info on this class

Reply:
Ok, so let's rename ElasticLog to something like `ElasticLogDestination`

Reply:
i renamed it to `ElasticClient`, because `ElasticDestination` already exists and implements `CiLoggerDestination`